.travis.yml

```diff
@@ -8,7 +14,7 @@ services:
 
 env:
   global:
-    - TERRAFORM_VERSION=0.9.4
+    - TERRAFORM_VERSION=0.11.10
```

Comment:
Upgrading to latest terraform version given deprecated features in previous one.